editor/tree_search.cpp

```diff
@@ -513,8 +513,8 @@ void TreeSearchPanel::_notification(int p_what) {
 
 void TreeSearchPanel::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("update_requested"));
```

Comment:
I am not sure whether we should also replace this with a LW_NAME entry.

Reply:
This is the only remaining hard-coded c-string.

Reply:
(For signals that is)

Reply:
Generally, in constructors and in `_bind_method`, we are not doing that. Unless in classes that are often created. It makes the most sense in places that are repeatedly executed.

Reply:
Do you want me to revert this?

Reply:
Not really, no.